samples/bot-suggested-actions/csharp/README.md

```diff
@@ -20,6 +20,7 @@ This sample shows the feature where user can [send suggested actions](https://le
 ## Included Features
 * Bots
 * Suggested Actions
+* Copilot Custom Engine Agents
 
 ## Interaction with app
```

Comment:
Gif file is not correct, All images of gif file should be equal size please refer other samples Gif Images

**Reference sample link :** 

https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-complete-sample/nodejs#interaction-with-app